Source/WebCore/platform/mac/WebNSAttributedStringExtras.mm

```diff
@@ -49,13 +49,13 @@
 
     attachmentRange = [originalString rangeOfString:attachmentCharString.get().get()];
     if (attachmentRange.location != NSNotFound && attachmentRange.length > 0) {
-        auto newAttributedString = adoptNS([attributedString mutableCopyWithZone:NULL]);
-        
+        NSMutableAttributedString *newAttributedString = [attributedString mutableCopy];
```

Comment:
Wes would adopt as soon as we create the copy. Not sure why you made this change. It is leak-prone to adopt later on.